Jcm5/edit show

jcm5 opened the pull request:
## Issue
Closes #84 #112 

## Description
Adds edit show functionality, allows for editing, adding, and deleting parameters

## Test Plan
Load a show with existing parameters
Add, edit, or delete parameters
Click apply changes to save parameters in a new file

## Expected Results
New file should be created that has the parameters listed/edited in the edit screen popup

# Style Score
8.88/10

## Bugs
Apply changes creates a new file, does not override existing

## Future Plans
Add OptionMenus instead of open text in editor popup

jcm5 committed:
Update gui.py

added edit show button and functionality

jcm5 committed:
Create edit_screen.py

class to edit shows

aholdeman reviewed (changes requested):
edit_screen.py needs to be moved to the djvj folder, not in the test folder

jcm5 committed:
Delete edit_screen.py

jcm5 committed:
Create edit_screen.py

jcm5 committed:
Merge branch 'master' into jcm5/edit_show

jcm5 committed:
Fixed error, clean up, method changes

UnboundLocalError: local variable 'item_list' referenced before assignment resolved
Removed unnecessary stuff
Fixed up remove_param
Changes to apply_changes
